node/node_test.go

```diff
@@ -136,18 +159,19 @@ func TestNodeSetAppVersion(t *testing.T) {
 }
 
 func TestNodeSetPrivValTCP(t *testing.T) {
+	t.Skip("TODO(ismail): Mock these conns using net.Pipe instead")
 	addr := "tcp://" + testFreeAddr(t)
 
 	config := cfg.ResetTestRoot("node_priv_val_tcp_test")
 	defer os.RemoveAll(config.RootDir)
 	config.BaseConfig.PrivValidatorListenAddr = addr
 
-	dialer := privval.DialTCPFn(addr, 100*time.Millisecond, ed25519.GenPrivKey())
+	dialer := privval.DialTCPFn(addr, 250*time.Millisecond, ed25519.GenPrivKey())
```

Comment:
OK, this change is actually not necessary!

```suggestion
	dialer := privval.DialTCPFn(addr, 100*time.Millisecond, ed25519.GenPrivKey())
```